src/Microsoft.Health.Dicom.Core.UnitTests/Features/Query/DicomQueryServiceTests.cs

```diff
@@ -24,7 +25,9 @@ public DicomQueryServiceTests()
             _queryService = new QueryService(
                 Substitute.For<IQueryParser>(),
                 Substitute.For<IQueryStore>(),
-                Substitute.For<IMetadataStore>());
+                Substitute.For<IMetadataStore>(),
+                Substitute.For<ICustomTagStore>(),
```

Comment:
We will need more UTs in general for all the new things that are added for QueryParser and how the dynamic SQL will be generated.

So try to separate it out with FF. So that we can turn off the feature if we want to